### ISSUE TITLE
__newindex on entities is not working correctly

### ISSUE THREAD

**Issue by CapsAdmin** [luastoned/LuaCraft#4]
it seems like you have to assign a value to the entity first, then after that __newindex works

**Comment by bkacjios** [comment]
```

**Comment by bkacjios** [comment]
```
[00:21:10] [Server thread/INFO] [LuaCraft]: Player433 Lua > LocalPlayer().test = "This is a test" print(LocalPlayer().test) 
This is a test
```

Should be fixed in commit f647941ff8098200fc403f986c51ba557b458876
